product_docs/docs/pgd/5/scaling.mdx

```diff
@@ -23,10 +23,10 @@ function to create or alter the definition of automatic range partitioning for a
 no definition exists, it's created. Otherwise, later executions will alter the
 definition.
 
-PGD Autopartition in PGD 5 currently locks the actual table while performing 
+PGD AutoPartition in PGD 5 currently locks the actual table while performing 
 new partition maintenance operations.
 
-An ERROR is raised if the table isn't RANGE partitioned or a multi-column
+An error is raised if the table isn't range partitioned or a multi-column
```

Comment:
Must range be all caps? I changed it here and below...I think it's probably okay like this?

Reply:
Should be ok. This is an old doc.